admin-core/src/main/resources/application.conf

```diff
@@ -74,4 +74,7 @@ drill {
         lifetime = "1h"
         lifetime = ${?DRILL_JWT_LIFETIME}
     }
+    users = [
+        "{\"username\": \"guest\", \"password\": \"guest\", \"role\": \"ADMIN\"}"
```

Comment:
lets just switch to "admin"/"admin" instead of "guest"/"guest" to clearly indicate level of default user access

Reply:
will be removed when db implemented